roles/kube_prometheus_stack/files/jsonnet/tests.yml

```diff
@@ -105,3 +105,18 @@ tests:
       - eval_time: 5m
         alertname: NodeTimeSkewDetected
         exp_alerts: []
+
+  - interval: 1m
+    input_series:
+      - series: 'mysql_up'
+        values: '1'
+    input_series:
+      - series: 'mysql_up'
+        values: '1'
+    input_series:
+      - series: 'mysql_up'
+        values: '0'                
```

Comment:
You're using the same key here 3 times, you need to have multiple series lines with perhaps different labels for the different instances.

You should have a test for no alerts if they're all up, and then alerts for the different ranges.

Reply:
already update the mysql_down unit test